nuxeo-runtime/nuxeo-runtime-test/src/main/java/org/nuxeo/runtime/test/runner/HotDeployer.java

```diff
@@ -28,11 +30,13 @@
  */
 public class HotDeployer {
 
-    public static final String DEPLOY_ACTION = "deploy";
+    protected static final String DEPLOY_ACTION = "deploy";
+
+    protected static final String UNDEPLOY_ACTION = "undeploy";
 
-    public static final String RESTART_ACTION = "restart";
+    protected static final String RESTART_ACTION = "restart";
 
-    public static final String RESET_ACTION = "reset";
+    protected static final String RESET_ACTION = "reset";
```

Comment:
Don't change visibility, we can need it in `HotDeployer.ActionHandler`.

Reply:
My bad, it was not used I changed it on a moment of weakness, back to public